service/src/main/java/bio/terra/drshub/services/AuthService.java

```diff
@@ -40,6 +41,12 @@ public class AuthService {
   private final Map<String, Optional<List<String>>> passportCache =
       Collections.synchronizedMap(new PassiveExpiringMap<>(1, TimeUnit.MINUTES));
 
+  private final Map<Pair<String, String>, SaKeyObject> serviceAccountKeyCache =
```

Comment:
can you comment what the key pair is here just for posterity

Reply:
Sure can!